### PR TITLE
VTCCA-3166: fixed back link issue found in testing

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala [hunk 1]
```diff
@@ -74,24 +75,44 @@ class ValuationsController @Inject()(
...
-                ),
-                owner = owner
-              ))
+          if (owner) {
+            Future.successful(
+              Ok(
+                views.html.dashboard.assessments(
+                  model = AssessmentsVM(
+                    assessmentsWithLinks = link.assessments
+                      .sortBy(_.currentFromDate.getOrElse(LocalDate.of(2017, 4, 7)))(
```
Comment (darrell-plant) [inline app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala:84]: We should factor this date out. Maybe config?

File: app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala [hunk 2]
```diff
@@ -137,7 +158,9 @@ class ValuationsController @Inject()(
       clientDetails match {
         case Some(client) =>
           config.newDashboardUrl(
-            s"selected-client-properties?clientOrganisationId=${client.organisationId}&clientName=${client.organisationName}&pageNumber=1&pageSize=15&sortField=ADDRESS&sortOrder=ASC")
+            s"selected-client-properties?clientOrganisationId=${client.organisationId}&clientName=${URLEncoder
+              .encode(client.organisationName, "UTF-8")}&pageNumber=1&pageSize=15&sortField=ADDRESS&sortOrder=ASC")
```
Comment (darrell-plant) [inline app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala:162]: are these the defaults (&pageNumber=1&pageSize=15&sortField=ADDRESS&sortOrder=ASC)
if so, we don't need them here.
I'm not sure if they are, but might simplify that code

**Comment by darrell-plant** [comment]
LGTM 👍 (with comments)

File: app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala (hunk 2 above)
Comment (habeeb01) [reply app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala:162]: yes they are defaults but unfortunately the link doesn't seem to work without them 🤷‍♂️ . That is the only reason why I added them

File: app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala (hunk 1 above)
Comment (habeeb01) [reply app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala:84]: Good point. I will do this in a separate commit

**Pull request merged**